index.d.ts

```diff
@@ -0,0 +1,52 @@
+// TypeScript typings for "mocha-testdata@1.1.3".
+
+declare module "mocha-testdata" {
```

Comment:
This still makes it an ambient module, doesn't it? I would expect that we just export the necessary functions and interfaces directly.